### PR TITLE
Cleared both statements on this article for Issue #1265

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solving Issue #1265, reviewed both statements and corrects the ones it was a copy/paste.

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Please do a CopyReview @nenonix @JohanFreelancer9  (thank you in advance!)

File: Exchange/ExchangeHybrid/hybrid-deployment/move-mailboxes.md [hunk 1]
```diff
@@ -70,9 +70,9 @@ You can use the remote move migration wizard on the **Office 365** tab in the EA
 ### Move on-premises mailboxes to Exchange Online
 <a name="bkmk_step3Move"> </a>
 
-You can use the remote move migration wizard on the **Office 365** tab in the EAC on an Exchange server to move existing user mailboxes in the on-premises organization to the Exchange Online organization. Follow these steps:
+You can use the remote move migration wizard on an Exchange server in the **Office 365** tab in the ECP to move existing user mailboxes in the on-premises organization to the Exchange Online organization. Follow these steps:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/move-mailboxes.md]:
```suggestion
You can use the remote move migration wizard in the **Office 365** tab on the ECP (Exchange Control Panel, On-premises) on an Exchange server to move existing user mailboxes in the on-premises organization to the Exchange Online organization. To do so, follow these steps:
```

File: Exchange/ExchangeHybrid/hybrid-deployment/move-mailboxes.md [hunk 2]
```diff
@@ -70,9 +70,9 @@ You can use the remote move migration wizard on the **Office 365** tab in the EA
...
 
-1. Open the EAC, and then navigate to **Office 365** \> **Recipients** \> **Migration**.
+1. Open the ECP (Exchange Control Panel, On-premises), and then navigate to the **Office 365** tab then **Recipients** \> **Migration**.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/move-mailboxes.md]:
```suggestion
1. Open the ECP, navigate to the **Office 365** tab and then **Recipients** \> **Migration**.
```

File: Exchange/ExchangeHybrid/hybrid-deployment/move-mailboxes.md [hunk 3]
```diff
@@ -103,7 +103,7 @@ You can use the remote move migration wizard on the **Office 365** tab in the EA
 ### Move Exchange Online mailboxes to the on-premises organization
 <a name="bkmk_eacExo"> </a>
 
-You can use the remote move migration wizard on the **Office 365** tab in the EAC on an Exchange server to move existing user mailboxes in the on-premises organization to the Exchange Online organization:
+You can use the remote move migration wizard on the **Office 365** tab in the EAC on Exchange Online to move existing cloud user mailboxes to the on-premises organization, to do so, please follow these steps:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-deployment/move-mailboxes.md]:
```suggestion
You can use the remote move migration wizard on the **Office 365** tab in the EAC on Exchange Online to move existing cloud user mailboxes to the on-premises organization. To do so, please follow these steps:
```

**Comment by chrisda** [comment]
Explain to me why ECP is better than EAC? Theoretically, this topic applies to Hybrid in all supported versions of Exchange Server. Only Exchange 2010 has ECP, and there are currently no other references to ECP aywhere in the topic.

**Comment by j0rt3g4** [comment]
@chrisda  There's not use of one over another one. What I said is basically if you go to "ECP/EAC" or whatever name the Control panel for exchange is. you can go on-premises and use the O365 tab to move mailboxes back and forth. That's it

**Comment by chrisda** [comment]
OK. I changed the ECP references to EAC.

**Comment by j0rt3g4** [comment]
Thank you @chrisda

**Pull request merged**